.github/workflows/conformance-aws-cni.yaml

```diff
@@ -290,6 +281,13 @@ jobs:
       - name: Post-test information gathering
         if: ${{ !success() }}
         run: |
+          echo "=== Install latest stable CLI ==="
```

Comment:
Why is this needed given that we are already installing the CLI in a previous step (which is also related to my other comment about downgrading it to `0.10`)?

Reply:
Ah, good point. Missed that part of the revert I guess.
I'd rather improve as a followup PR (which will be needed anyway) than to have to rerun the tests 10 times here, again.